apps/server/src/modules/files/service/files.service.ts

```diff
@@ -1,137 +1,18 @@
-	}
-
-	async removeUserPermissionsOrCreatorReferenceToAnyFiles(userId: EntityId): Promise<DomainDeletionReport> {
-		this.logger.info(
-			new DataDeletionDomainOperationLoggable(
-				'Deleting user data from Files',
-				DomainName.FILE,
-				userId,
-				StatusModel.PENDING
-			)
-		);
-		const entities = await this.repo.findByPermissionRefIdOrCreatorId(userId);
-
-		entities.forEach((entity) => {
-			entity.removePermissionsByRefId(userId);
-			entity.removeCreatorId(userId);
-		});
-
-		await this.repo.save(entities);
-
-		const numberOfUpdatedFiles = entities.length;
-
-		const result = DomainDeletionReportBuilder.build(DomainName.FILE, [
-			DomainOperationReportBuilder.build(OperationType.UPDATE, numberOfUpdatedFiles, this.getFilesId(entities)),
-		]);
-
-		this.logger.info(
-			new DataDeletionDomainOperationLoggable(
-				'Successfully removed user data from Files',
-				DomainName.FILE,
-				userId,
-				StatusModel.FINISHED,
-				numberOfUpdatedFiles,
-				0
-			)
-		);
-
-		return result;
-	}
-
-	async findFilesOwnedByUser(userId: EntityId): Promise<FileEntity[]> {
-		return this.repo.findByOwnerUserId(userId);
 	}
 
-	async markFilesOwnedByUserForDeletion(userId: EntityId): Promise<DomainDeletionReport> {
-		this.logger.info(
-			new DataDeletionDomainOperationLoggable(
-				'Marking user files to deletion',
-				DomainName.FILE,
-				userId,
-				StatusModel.PENDING
-			)
-		);
-		const entities = await this.repo.findByOwnerUserId(userId);
-
-		entities.forEach((entity) => entity.markForDeletion());
-
-		await this.repo.save(entities);
-
-		const numberOfMarkedForDeletionFiles = entities.length;
-
-		const result = DomainDeletionReportBuilder.build(DomainName.FILE, [
-			DomainOperationReportBuilder.build(
-				OperationType.UPDATE,
-				numberOfMarkedForDeletionFiles,
-				this.getFilesId(entities)
-			),
-		]);
-
-		this.logger.info(
-			new DataDeletionDomainOperationLoggable(
-				'Successfully marked user files for deletion',
-				DomainName.FILE,
-				userId,
-				StatusModel.FINISHED,
-				numberOfMarkedForDeletionFiles,
-				0
-			)
-		);
-
-		return result;
-	}
-
-	public async deleteUserData(userId: EntityId): Promise<DomainDeletionReport> {
-		const [markedFilesForDeletion, removedUserPermissionsToFiles] = await Promise.all([
-			this.markFilesOwnedByUserForDeletion(userId),
-			this.removeUserPermissionsOrCreatorReferenceToAnyFiles(userId),
-		]);
-
-		const modifiedFilesCount =
-			markedFilesForDeletion.operations[0].count + removedUserPermissionsToFiles.operations[0].count;
-		const modifiedFilesRef = [
-			...markedFilesForDeletion.operations[0].refs,
-			...removedUserPermissionsToFiles.operations[0].refs,
-		];
-
-		const result = DomainDeletionReportBuilder.build(DomainName.FILE, [
-			DomainOperationReportBuilder.build(OperationType.UPDATE, modifiedFilesCount, modifiedFilesRef),
-		]);
-
-		return result;
-	}
+	public async findFilesAccessibleOrCreatedByUser(userId: EntityId): Promise<FileEntity[]> {
```

Comment:
I see this method is only used in tests. It makes no sense.
The whole file should be removed...

Reply:
done.